bugbug/models/bug.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, lemmatization=False):
             bug_features.affected_then_unaffected(),
             bug_features.product(),
             bug_features.component(),
+            bug_features.number_of_bug_dependencies(),
```

Comment:
Since there was no improvement for this model, we shouldn't add the feature here.